packages/shopify-codemod/test/fixtures/mocha-context-to-closure/contextually-declared-functions.input.js

```diff
@@ -0,0 +1,12 @@
+suite('top-level', function() {
+  setup('name', () => {
+    this.foo = 'foo';
+    this.bar = function() {
+      return this.foo === 'foo';
```

Comment:
Wait yes it should be, because in this case the coffeescript code would have `=>` which is converted to arrow func, so this only needs to operate on those